README.md

```diff
@@ -71,8 +71,40 @@ Pravega operator can be upgraded by modifying the image tag using
 ```
 $ kubectl edit <operator deployment name>
 ```
-Currently, the minor version upgrade is supported, e.g. 0.4.0 -> 0.4.1. However, the major version upgrade
-has not been supported yet, e.g. 0.4.0 -> 0.5.0.
+This approach will work for upgrades upto 0.4.0
+
+Starting from operator version 0.5 onwards, pravega operator is not handling bookies, and that will be handled by bookkeeper-operator.So while upgrading from 0.4 to 0.5, we have to tansfer the ownership of bookkeeper objects to bookkeeper operator. For this we are maintaining 2 versions namely, v1alpha1 and v1beta1 inside the crd. v1alpha1 is for older versions and v1beta1 is for newer versions. And during upgrade we are triggering conversion webhook that will change the current version to v1beta1
```

Comment:
Lets say 
v1alpha1 == Pravega Custom Resource with Bookkeeper
v1beta1 == Pravega Custom Resource without Bookkeeper

also need to mention other changes in v1beta1 like change in name of tier2 and addition of BookkeeperUri.